recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ requirements:
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
     - setuptools
+    - libgcc  # [linux]
```

Comment:
So we have had a lot of problems when adding `libgcc`. Namely some things get linked against system libraries when built and some things get linked to the conda supplied libraries. Hunting these issues down and resolving them was an endless source of headaches. Please see issue ( https://github.com/conda-forge/matplotlib-feedstock/issues/32 ) for the effects on `matplotlib`. However, these issues happened throughout our stack. Not inclined to go back down that road.